src/appsignal/opentelemetry.py

```diff
@@ -80,7 +80,9 @@ def add_requests_instrumentation() -> None:
 
 DefaultInstrumentationAdder = Callable[[], None]
 
-DEFAULT_INSTRUMENTATION_ADDERS = {
+DEFAULT_INSTRUMENTATION_ADDERS: Dict[
```

Comment:
```suggestion
DEFAULT_INSTRUMENTATION_ADDERS: dict[
```

Reply:
Or better yet, use `Mapping`, then mypy will make sure nobody modifies the constant.

Reply:
💡 I like that a lot. Will do, thanks!